src/pds_doi_service/core/db/transaction_on_disk.py

```diff
@@ -138,25 +139,30 @@ def write(self, transaction_dir, input_ref=None, output_content=None, output_con
             The content type of output_content. Should be one of "xml" or "json".
 
         """
-
-        # Set up the appropriate umask in-case os.makedirs needs to create any
-        # intermediate parent directories (its mask arg only affects the created leaf directory)
+        """
+        Set up the appropriate umask in-case os.makedirs needs to create any
+        intermediate parent directories (its mask arg only affects the created leaf directory)
+        """
         prev_umask = os.umask(0o0002)
 
-        # Create the new transaction history directory with group-rw enabled
+        """ Create the new transaction history directory with group-rw enabled """
         os.makedirs(transaction_dir, exist_ok=True, mode=0o0775)
 
         if input_ref:
             if os.path.isdir(input_ref):
-                # Copy the input files, but do not preserve their permissions so
-                # the umask we set above takes precedence
+                """
```

Comment:
@rsjoyner in the future. these should be commented out with `#` the multi-line comments should only be used in docstring. I will fix this.